### PR TITLE
Rewrite truncate tests

### LINKED ISSUE #550: Rewrite truncate tests using jest
Rewrite https://github.com/airbrake/airbrake-js/blob/master/test/unit/truncate_test.ts using Jest in https://github.com/airbrake/airbrake-js/tree/master/tests. See https://github.com/airbrake/airbrake-js/blob/master/tests/client.test.js for an example.

### PR DESCRIPTION
**Pull request by anmic**
Fixes #550.
I rewrote tests.
You can run the test by using command: `npm run test`

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -162,7 +162,10 @@ module.exports = {
...
 
   // A map from regular expressions to paths to transformers
-  // transform: null,
+  "transform": {
+    "^.+\\.tsx?$": "ts-jest",
+    "^.+\\.js$": "babel-jest"
+  },
 
   // An array of regexp pattern strings that are matched against all source file paths, matched files will skip transformation
   // transformIgnorePatterns: [
```
Comment (vmihailenco) [inline jest.config.js]: revert all changes in this file

File: package.json [hunk 2]
```diff
@@ -45,6 +45,7 @@
     "sinon": "7.3.2",
     "sinon-chai": "3.3.0",
     "source-map-loader": "0.2.4",
+    "ts-jest": "^24.0.2",
```
Comment (vmihailenco) [inline package.json]: remove

File: src/client.ts [hunk 3]
```diff
@@ -21,7 +21,7 @@ import { Historian } from './historian';
 import Options from './options';
 import { IRequestInfo, Routes } from './routes';
 
-declare const VERSION: string;
+const VERSION = '1.6.8';
```
Comment (vmihailenco) [inline src/client.ts]: revert

**Comment by anmic** [comment]
I reverted modifications in config files.

**Pull request merged**